src/Services/DatabaseTools/AbstractDatabaseTool.php

```diff
@@ -248,7 +248,7 @@ protected function getMetadatas(): array
         if (!isset(self::$cachedMetadatas[$key])) {
             self::$cachedMetadatas[$key] = $this->om->getMetadataFactory()->getAllMetadata();
             usort(self::$cachedMetadatas[$key], function ($a, $b) {
-                return strcmp($a->name, $b->name);
+                return strcmp($a->getName(), $b->getName());
```

Comment:
It looks like this code is never called. Adding a typo there doesn't break the tests.

Reply:
`end()` is used to return the parent. While this looks weird, it's not necessary to end with `end()`. I'll revert this change to avoid regressions.